west.yml

```diff
@@ -57,7 +59,8 @@ manifest:
       groups:
         - hal
     - name: hal_espressif
-      revision: 8c7ee781ea4c4a010740f5231709ce7b21ad96d6
+      remote: marekmatej
```

Comment:
Update to the proper `hal_espressif` revision.

Reply:
right!